packages/edit-post/src/components/keyboard-shortcut-help-modal/config.js

```diff
@@ -36,4 +36,14 @@ export const textFormattingShortcuts = [
 		keyCombination: { modifier: 'access', character: 'x' },
 		description: __( 'Make the selected text inline code.' ),
 	},
+	{
+		keyCombination: { modifier: 'access', character: 'Level' },
+		description: __(
+			'Convert the current paragraph to a heading of level 1-6.'
+		),
+	},
+	{
+		keyCombination: { modifier: 'access', character: '0' },
```

Comment:
Nitpick, I think this could be put above the other shortcut, so that it's in numerical order (0 and then 1-6)